crates/wash-lib/src/parser/mod.rs

```diff
@@ -941,7 +685,9 @@ impl TryFrom<RawDevConfig> for DevConfig {
 /// * `opt_path` - The path to the config file. If None, it will look for a wasmcloud.toml file in the current directory.
 /// * `use_env` - Whether to use the environment variables or not. If false, it will not attempt to use environment variables. Defaults to true.
 pub fn get_config(opt_path: Option<PathBuf>, use_env: Option<bool>) -> Result<ProjectConfig> {
```

Comment:
Super nit: This should be a method called "load" or something similar on `ProjectConfig`. Just calling that out since we're in refactor land here